.travis.yml

```diff
@@ -66,6 +71,13 @@ matrix:
           python: "3.7"
           env: BUILD=tests
 
+        - os: linux
+          dist: xenial
+          sudo: required
```

Comment:
Hello @1st1,
`sudo` option [is deprecated](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).
```suggestion
```

Reply:
@1st1 I see you removed `sudo` options.
Just want to note that one of them [is still here](https://github.com/MagicStack/uvloop/blob/0c2cb54dbbf7a7661b912e76e2ae7d35e68e1922/.travis.yml#L67).

Reply:
Thanks!